conf/machine/raspberrypi0-2w.conf

```diff
@@ -0,0 +1,17 @@
+#@TYPE: Machine
+#@NAME: RaspberryPi0 2 Wifi Development Board (32bit)
+#@DESCRIPTION: Machine configuration for the RaspberryPi0 2 Wifi in 32 bits mode
+
+include conf/machine/raspberrypi3.conf
+
+MACHINEOVERRIDES := "${@'${MACHINEOVERRIDES}'.replace(':${MACHINE}',':raspberrypi3:${MACHINE}')}"
+
+MACHINE_EXTRA_RRECOMMENDS += "\
+    linux-firmware-rpidistro-bcm43436 \
+    linux-firmware-rpidistro-bcm43436s \
+    bluez-firmware-rpidistro-bcm43430b0-hcd \
+"
+
+RPI_KERNEL_DEVICETREE = " \
+    broadcom/bcm2710-rpi-zero-2.dtb \
```

Comment:
This won't work on 32 bit.

Reply:
Ok, I removed it.

Reply:
Have you tried to build the 32 machine?  I'm wondering if the default value of `RPI_KERNEL_DEVICETREE` makes any sense anymore because the arm and arm64 have now different dtbs including different path.

Reply:
No, we're only interested in the 64bit image

Reply:
If you only have only tested the 64 variant, I'd say to only add that for now. I'm not sure if the 32 one builds now.

Reply:
Sure, I removed it.